src/controllers/playback/video/index.html

```diff
@@ -35,6 +35,10 @@ <h3 class="osdTitle"></h3>
                     <span class="xlargePaperIconButton material-icons skip_previous"></span>
                 </button>
 
+                <button is="paper-icon-button-light" class="btnPreviousChapter autoSize hide" title="${PreviousChapter}">
```

Comment:
Currently these buttons are always hidden because the `hide` class is never removed. I'm not sure if it would make sense to always display these buttons, or if they should only be displayed under certain conditions.

Reply:
I guess ideally they would only be shown when the video file has chapters.

Reply:
I've now added functionality for the buttons to only be shown if the item has chapters.

Reply:
Also, its rebased now.